### ISSUE TITLE
Internationalise verification emoji based on the language of the person you're verifying

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#9298]
In a world where we can advertise the languages we speak in our extensible profile, it'd be lovely for the SAS emoji to get labelled both in your local language as well as the remote one, so you don't have to remember how to translate Unicorn into French

**Comment by exyi** [comment]
I agree, the language mismatch is actually quite annoying when in dealing with Czech people having Riot in English. You have to translate the words at some point, which is less smooth than not having to do so.

> so you don't have to remember how to translate Unicorn into French

Sometimes, people don't have a idea how to translate anything into French and the French guy also don't have a clue about Czech. In that case you have to communicate everything though English, which is quite losy (esp. when our English skill are questionable)

**Comment by ShadowJonathan** [comment]
Isn't it a privacy hazard to advocate the language of the local user to the other user? Unsure how useful this is \w pros and cons. This needs Product

**Comment by t3chguy** [comment]
And spec to share the language during verification
